scripts/exglobal_atmos_analysis.sh

```diff
@@ -747,7 +747,7 @@ cat > gsiparm.anl << EOF
   diag_precon=.true.,step_start=1.e-3,emiss_bc=.true.,nhr_obsbin=${nhr_obsbin:-3},
   cwoption=3,imp_physics=${imp_physics},lupp=${lupp},cnvw_option=${cnvw_option},cao_check=${cao_check},
   netcdf_diag=${netcdf_diag},binary_diag=${binary_diag},
-  lobsdiag_forenkf=${lobsdiag_forenkf},
+  lobsdiag_forenkf=${lobsdiag_forenkf}
```

Comment:
Missing a comma here
```suggestion
  lobsdiag_forenkf=${lobsdiag_forenkf},
```

Reply:
Done